.github/workflows/production_e2e.yml

```diff
@@ -0,0 +1,57 @@
+name: Production E2E Tests
+on:
+  schedule:
+    - cron: '0 * * * *' # Run every hour
+  workflow_dispatch:
+env:
+  # Playwright headless browsers running in CI get low confidence scores, causing flaky e2e tests. Lower the confidence score threshold for CI testing.
+  MIN_CONFIDENCE_SCORE: 0
+  # Run e2e tests against production
+  PRODUCTION_E2E_TEST_BASE_URL: 'https://demo.fingerprint.com'
+
+jobs:
+  e2e:
+    name: Playwright
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Check please the latest available versions of these default actions. It will help to fix [warnings](https://github.com/fingerprintjs/fingerprintjs-pro-use-cases/actions/runs/7670180556) bu using node20

Reply:
Good point, done, thanks! https://github.com/fingerprintjs/fingerprintjs-pro-use-cases/pull/116/commits/393a43449eee944c4d7e5f75a075f3d5282792c7